k8s/neuvector/neuvector.yaml

```diff
@@ -12,8 +12,10 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: neuvector-azure-keyvault
-    version: 0.0.5
+    version: 0.0.6
   values:
+    config:
+      autoScan: true
```

Comment:
I thought you set this to true by default?

Reply:
it's overridden by kustomise for what I could see in perftest (see previous PRs)